helm/flowforge/templates/broker.yaml

```diff
@@ -146,11 +144,5 @@ spec:
     protocol: TCP
     name: mqtt-ws
   selector:
-    {{- if and ( eq .Values.forge.broker.enabled true) ( eq .Values.forge.broker.teamBroker.enabled false ) -}}
     {{- include "forge.brokerSelectorLabels" . | nindent 4 }}
-    {{- else -}}
-    apps.emqx.io/db-role: core
-    apps.emqx.io/instance: emqx
-    apps.emqx.io/managed-by: emqx-operator
-    {{- end }}
```

Comment:
This one overwrites the quick fix we made yestarday.

Reply:
doh, must have forgot to pull first, will rebase